openretina/models/core_readout.py

```diff
@@ -25,6 +25,7 @@ def __init__(
         learning_rate: float,
         loss: nn.Module | None = None,
         correlation_loss: nn.Module | None = None,
+        data_info: dict[str, int] | None = None,
```

Comment:
```suggestion
        data_info: dict[str, Any] | None = None,
```

I think the int is wrong in the type annotation